api_app/api.py

```diff
@@ -199,6 +287,95 @@ def ask_analysis_availability(request):
             type=OpenApiTypes.STR,
             description="optional if is_sample=True, the binary name",
         ),
+        OpenApiParameter(
+            name="analyzers_requested",
+            type=OpenApiTypes.OBJECT,
+            description="list of requested analyzer to run, before filters",
+        ),
+        OpenApiParameter(
+            name="tags_id",
+            type=OpenApiTypes.OBJECT,
+            description="list of id's of tags to apply to job",
+        ),
+        OpenApiParameter(
+            name="run_all_available_analyzers",
+            type=OpenApiTypes.BOOL,
+            description="default False",
+        ),
+        OpenApiParameter(
+            name="private",
+            type=OpenApiTypes.BOOL,
+            description="""
+            Default False,
+            enable it to allow view permissions to only requesting user's groups.""",
+        ),
+        OpenApiParameter(
+            name="force_privacy",
+            type=OpenApiTypes.BOOL,
+            description="""
+            Default False,
+            enable it if you want to avoid to run analyzers with privacy issues""",
+        ),
+        OpenApiParameter(
+            name="disable_external_analyzers",
+            type=OpenApiTypes.BOOL,
+            description="""
+            Default False, enable it if you want to exclude external analyzers""",
+        ),
+        OpenApiParameter(
+            name="runtime_configuration",
+            type=OpenApiTypes.OBJECT,
+            description=r"""
+            Default {}, contains additional parameters for particular analyzers""",
+        ),
+        OpenApiParameter(
+            name="test",
+            type=OpenApiTypes.BOOL,
+            description="disable analysis for API testing",
+        ),
+    ],
+    responses={
```

Comment:
We do not need to specify the parameters now, set this like: `request=serializers.FileAnalysisSerializer` (refer to: vhttps://drf-spectacular.readthedocs.io/en/latest/customization.html#step-2-extend-schema)

Reply:
same for the `analyze_observable`.